Carbon.IIS/Functions/Install-CIisAppPool.ps1

```diff
@@ -8,12 +8,9 @@ function Install-CIisAppPool
     .DESCRIPTION
     The `Install-CIisAppPool` function creates or updates an IIS application pool. Pass the name of the application pool
     to the `Name` parameter. If that application pool doesn't exist, it is created. If it does exist, its configuration
-    is updated to match the values of the arguments passed.
-
-    If you pass just a name, the function creates a 64-bit application pool that runs an integrated pipeline
-    using .NET 4.0 managed runtime. To use a 32-bit app pool, set the `Enable32BitAppOnWin64` parameter to `$true`. To use a
-    classic pipeline, set the `ManagedPipelineMode` parameter to `Classic`. To use a different version of .NET, use the
-    `ManagedRuntimeVersion` parameter.
+    is updated to match the values of the arguments passed. If you don't pass an argument, that argument's setting is
+    deleted and reset to its default value. You always get an application pool with the exact same configuration, even

```

Comment:
"deleted and reset to its default value" appears to disagree with `-Reset` notes.

Reply:
Changed to "deleted, which resets it to its default value". I like that wording better, so updated all the `-Reset` docs to match.